frontends/concrete-python/tests/representation/test_node.py

```diff
@@ -38,6 +38,32 @@ def test_node_bad_constant(constant, expected_error, expected_message):
     assert str(excinfo.value) == expected_message
 
 
+def default_tfhers_dtype() -> tfhers.TFHERSIntegerType:
+    """Get default tfhers type used for testing.
+
+    Returns:
+        tfhers.TFHERSIntegerType: default type for testing
+    """
+    tfhers_params = tfhers.TFHERSParams(
+        761,
+        1,
+        2048,
+        6.36835566258815e-06,
+        3.1529322391500584e-16,
+        23,
+        1,
+        3,
+        5,
+        4,
+        4,
+        5,
+        -40.05,
+        None,
+        True,
+    )
+    return tfhers.int8_2_2(tfhers_params)
```

Comment:
Can this be a constant?

Reply:
It can yes, but it will means being loaded whenever tests are executed, even if it's not part of selected tests. Having it as a function makes it used in specific tests only

Reply:
Sure.